### DISCUSSION TITLE
recipes/breads/japanese-milk-bread/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#1034] [Comments]
# recipes/breads/japanese-milk-bread/

A collection of my recipes.

https://nicholaswilde.io/recipes/breads/japanese-milk-bread/

**Comment by nicholaswilde** [comment]
Used canola oil but bread stuck to pan. Use butter or line with parchment instead.
Bread broke a part in two when removed from pan.
A bunch of air pockets on bottom of loaf. Pack dough into pan after shaping.
During divide, make 4 balls and rest covered for 15 minutes to relax dough, then shape and roll.
